src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,38 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
 public class EventManager {
+    private ExecutorService executor;
+    private List<EventListener> listeners = new CopyOnWriteArrayList<>();
+
+    public EventManager() {
+        executor = Executors.newFixedThreadPool(5);
```

Comment:
Consider initializing the ExecutorService within an instance initialization block or static block to ensure it's only created once and not on every instance creation. If it's intended for use across multiple instances, it should be static.